addons/account_payment/tests/test_payment_provider.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.addons.account_payment.tests.common import AccountPaymentCommon
+from odoo.tests import tagged
```

Comment:
In Odoo, imports should be grouped as follows: standard libs, external libs, ORM imports, addon imports.
```suggestion
from odoo.tests import tagged

from odoo.addons.account_payment.tests.common import AccountPaymentCommon
```